config/common/common.go

```diff
@@ -41,6 +41,15 @@ const (
 	nameReferenceKind = "name"
 	// idReferenceKind represents ID reference kind
 	idReferenceKind = "id"
+
+	// VersionV1Alpha2 is used as minimum version for all manually configured resources.
+	VersionV1Alpha2 = "v1alpha2"
```

Comment:
We may consider moving these constants to `apis/rconfig` package where `APISPackagePath` resides, as these are API type constants.

Reply:
Moved to rconfig package!